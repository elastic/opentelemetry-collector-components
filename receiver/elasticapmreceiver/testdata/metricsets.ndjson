{"metadata": {"user": {"username": "logged-in-user", "id": "axb123hg", "email": "user@mail.com"}, "labels": {"tag0": null, "tag1": "one", "tag2": 2}, "process": {"ppid": null, "pid": 1234, "argv": null, "title": null}, "system": null, "service": {"name": "1234_service-12a3", "node": {"configured_name": "node-1"},"language": {"version": null, "name":"ecmascript"}, "agent": {"version": "3.14.0", "name": "elastic-node", "activation_method": "some_activation_method"}, "environment": null, "framework": null,"version": null, "runtime": null}}}
{"metricset": { "samples": { "transaction.breakdown.count":{"value":12}, "transaction.duration.sum.us":{"value":12}, "transaction.duration.count":{"value":2}, "transaction.self_time.sum.us":{"value":10}, "transaction.self_time.count":{"value":2}, "span.self_time.count":{"value":1},"span.self_time.sum.us":{"value":633.288}, "extraneous_metrics_ignored": { "value": -1022 } }, "tags": { "some": "abc", "code": 200, "success": true }, "transaction":{"type":"request","name":"GET /"},"span":{"type":"db","subtype":"mysql"},"timestamp": 1496170422281000 }}