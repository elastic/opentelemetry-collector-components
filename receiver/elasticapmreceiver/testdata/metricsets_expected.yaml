resourceMetrics:
  - resource:
      attributes:
        - key: service.name
          value:
            stringValue: 1234_service-12a3
        - key: service.version
          value:
            stringValue: ""
        - key: telemetry.sdk.language
          value:
            stringValue: ecmascript
        - key: telemetry.sdk.name
          value:
            stringValue: ElasticAPM
    scopeMetrics:
      - metrics:
          - gauge:
              dataPoints:
                - asInt: "633288"
                  attributes:
                    - key: processor.event
                      value:
                        stringValue: metric
                    - key: span.subtype
                      value:
                        stringValue: mysql
                    - key: span.type
                      value:
                        stringValue: db
                    - key: transaction.name
                      value:
                        stringValue: GET /
                    - key: transaction.type
                      value:
                        stringValue: request
                  timeUnixNano: "1496170422281000000"
            name: span.self_time.sum.us
            unit: us
          - gauge:
              dataPoints:
                - asDouble: 1
                  attributes:
                    - key: processor.event
                      value:
                        stringValue: metric
                    - key: span.subtype
                      value:
                        stringValue: mysql
                    - key: span.type
                      value:
                        stringValue: db
                    - key: transaction.name
                      value:
                        stringValue: GET /
                    - key: transaction.type
                      value:
                        stringValue: request
                  timeUnixNano: "1496170422281000000"
            name: span.self_time.count
            unit: '{span}'
        scope: {}
  - resource:
      attributes:
        - key: service.name
          value:
            stringValue: 1234_service-12a3
        - key: service.version
          value:
            stringValue: ""
        - key: telemetry.sdk.language
          value:
            stringValue: ecmascript
        - key: telemetry.sdk.name
          value:
            stringValue: ElasticAPM
    scopeMetrics:
      - metrics:
          - gauge:
              dataPoints:
                - asDouble: 6.520832e+06
                  attributes:
                    - key: event.outcome
                      value:
                        stringValue: unknown
                    - key: processor.event
                      value:
                        stringValue: metric
                    - key: timestamp.us
                      value:
                        intValue: "1496170421364000"
                  timeUnixNano: "1496170421364000000"
            name: go.memstats.heap.sys.bytes
        scope: {}
  - resource:
      attributes:
        - key: service.name
          value:
            stringValue: 1234_service-12a3
        - key: service.version
          value:
            stringValue: ""
        - key: telemetry.sdk.language
          value:
            stringValue: ecmascript
        - key: telemetry.sdk.name
          value:
            stringValue: ElasticAPM
    scopeMetrics:
      - metrics:
          - gauge:
              dataPoints:
                - asDouble: 2048
                  attributes:
                    - key: event.outcome
                      value:
                        stringValue: unknown
                    - key: processor.event
                      value:
                        stringValue: metric
                    - key: timestamp.us
                      value:
                        intValue: "1496170421366000"
                  timeUnixNano: "1496170421366000000"
            name: system.process.cgroup.memory.mem.limit.bytes
          - gauge:
              dataPoints:
                - asDouble: 1024
                  attributes:
                    - key: event.outcome
                      value:
                        stringValue: unknown
                    - key: processor.event
                      value:
                        stringValue: metric
                    - key: timestamp.us
                      value:
                        intValue: "1496170421366000"
                  timeUnixNano: "1496170421366000000"
            name: system.process.cgroup.memory.mem.usage.bytes
        scope: {}
  - resource:
      attributes:
        - key: service.name
          value:
            stringValue: 1234_service-12a3
        - key: service.version
          value:
            stringValue: ""
        - key: telemetry.sdk.language
          value:
            stringValue: ecmascript
        - key: telemetry.sdk.name
          value:
            stringValue: ElasticAPM
    scopeMetrics:
      - metrics:
          - gauge:
              dataPoints:
                - asDouble: 2048
                  attributes:
                    - key: event.outcome
                      value:
                        stringValue: unknown
                    - key: processor.event
                      value:
                        stringValue: metric
                    - key: timestamp.us
                      value:
                        intValue: "1496170421367000"
                  timeUnixNano: "1496170421367000000"
            name: system.process.cgroup.cpu.stats.throttled.ns
          - gauge:
              dataPoints:
                - asDouble: 2048
                  attributes:
                    - key: event.outcome
                      value:
                        stringValue: unknown
                    - key: processor.event
                      value:
                        stringValue: metric
                    - key: timestamp.us
                      value:
                        intValue: "1496170421367000"
                  timeUnixNano: "1496170421367000000"
            name: system.process.cgroup.cpuacct.id
          - gauge:
              dataPoints:
                - asDouble: 2048
                  attributes:
                    - key: event.outcome
                      value:
                        stringValue: unknown
                    - key: processor.event
                      value:
                        stringValue: metric
                    - key: timestamp.us
                      value:
                        intValue: "1496170421367000"
                  timeUnixNano: "1496170421367000000"
            name: system.process.cgroup.cpuacct.total.ns
          - gauge:
              dataPoints:
                - asDouble: 1024
                  attributes:
                    - key: event.outcome
                      value:
                        stringValue: unknown
                    - key: processor.event
                      value:
                        stringValue: metric
                    - key: timestamp.us
                      value:
                        intValue: "1496170421367000"
                  timeUnixNano: "1496170421367000000"
            name: system.process.cgroup.cpu.cfs.period.us
          - gauge:
              dataPoints:
                - asDouble: 2048
                  attributes:
                    - key: event.outcome
                      value:
                        stringValue: unknown
                    - key: processor.event
                      value:
                        stringValue: metric
                    - key: timestamp.us
                      value:
                        intValue: "1496170421367000"
                  timeUnixNano: "1496170421367000000"
            name: system.process.cgroup.cpu.id
          - gauge:
              dataPoints:
                - asDouble: 2048
                  attributes:
                    - key: event.outcome
                      value:
                        stringValue: unknown
                    - key: processor.event
                      value:
                        stringValue: metric
                    - key: timestamp.us
                      value:
                        intValue: "1496170421367000"
                  timeUnixNano: "1496170421367000000"
            name: system.process.cgroup.cpu.cfs.quota.us
          - gauge:
              dataPoints:
                - asDouble: 2048
                  attributes:
                    - key: event.outcome
                      value:
                        stringValue: unknown
                    - key: processor.event
                      value:
                        stringValue: metric
                    - key: timestamp.us
                      value:
                        intValue: "1496170421367000"
                  timeUnixNano: "1496170421367000000"
            name: system.process.cgroup.cpu.stats.periods
          - gauge:
              dataPoints:
                - asDouble: 2048
                  attributes:
                    - key: event.outcome
                      value:
                        stringValue: unknown
                    - key: processor.event
                      value:
                        stringValue: metric
                    - key: timestamp.us
                      value:
                        intValue: "1496170421367000"
                  timeUnixNano: "1496170421367000000"
            name: system.process.cgroup.cpu.stats.throttled.periods
        scope: {}
